stack.yaml

```diff
@@ -23,7 +23,7 @@ packages:
   extra-dep: true
 - location:
     git: https://github.com/serokell/kademlia.git
-    commit: b4b0a6774368b9bbea96d7a03a1a451b6fc3cfea
+    commit: 2a51c89d39faef920d11137490294f34a8969601
```

Comment:
Don't forget to update this commit & one for time-warp after PR for kademlia gets merged to master